docs/source/how_to/use_oauth2.rst

```diff
@@ -147,9 +147,59 @@ However, if the response is not successful, you’ll receive an error response.
 Using the access token
 ----------------------
 
-.. note::
+Now that you have the access token, you can get the user data by sending a
+``GET`` request to ``/open_auth/get_user_data/`` with the access token in the
+header as follows:
+
+.. code-block:: json
+
+  {
+    "Authorization": "Bearer vqygrcouTuyAYHZLz3rGcZf5FpPd3K"
+  }
+
+where ``vqygrcouTuyAYHZLz3rGcZf5FpPd3K`` is the access token.
+
+If the access token is valid, then you will receive a 200 response
+with a dictionary containing the user's information based on the scopes
+of the client app, similar to this:
+
+.. code-block:: json
```

Comment:
Is this JSON taken directly from an endpoint output? Because the spacing around the `:` is inconsistent.

Reply:
Yeah, I copy-pasted the response and then deleted the actual information with dummy values, that's when the spacing must've changed..

Reply:
Please ensure a space after the `:`.

Reply:
Okay 👍